src/components/config-sessions.tsx

```
@@ -0,0 +1,407 @@
+                  setFilter(k)
+                }
+              }}
+            >
+              {t(`sessions.filter.${k}`)}
+              {' '}
+              ({counts[k as keyof typeof counts]})
+            </Chip>
+          ))}
+          <div className="ml-auto flex items-center gap-1">
+            <span className="text-xs text-muted">{t('sessions.sort.label')}</span>
+            <Button size="sm" variant={sortKey === 'createdAt' ? 'primary' : 'tertiary'} className="h-7 rounded-md text-xs" onPress={() => { setSortKey('createdAt'); setSortAsc(v => sortKey === 'createdAt' ? !v : false) }}>
```

ESLint

[error] This line has 2 statements. Maximum allowed is 1.

(style/max-statements-per-line)